src/svg-icons/Icon.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react'
+import cn from 'classnames'
+import getColored from './getColored'
+import injectSheet from '../jss'
+
+const Icon = ({sheet: {classes}, className, name, color = '#000'}) => (
+  <img
```

Comment:
fyi unless we specify an alt text or aria-label this should be hidden by screen readers. 
You don't have to document yourself on this – it is a fact.

The component should get a `prop` called `label` which by default is undefined.

When a label is provided when we set the alt `<img src={...} alt={label} />` otherwise we add `aria-hidden="true"`: `<img src={...} alt aria-hidden="true" />`